timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -219,26 +220,20 @@ void declare_context(py::module &m) {
                 if (burn_in < 0) {
                     throw std::runtime_error("burn in steps must be greater than zero");
```

Comment:
nit[edge-case] `burn_in <= 0` to match error message, or update error message to match `burn_in < 0`

Reply:
Good catch, fixed in [6a8044f](https://github.com/proteneer/timemachine/pull/981/commits/6a8044fb14740fd3aa406915d5278136d11e3f12)